ssg/playbook_builder.py

```diff
@@ -158,6 +158,10 @@ def create_playbook(self, snippet_path, rule_id, variables,
                 (rule_id, snippet_path)
             )
 
+        tags = set()
+        for task in play_tasks:
+            tags |= set(task.pop("tags", []))
+
```

Comment:
It would be interesting to have a test for this for a rule with multiple tasks. For example Ansible fix for audit rule privileged commands.

Reply:
Filed https://github.com/ComplianceAsCode/content/issues/4037 regarding this.